demo/vmdefs/available/centos6-wordpress/Vagrantfile

```diff
@@ -0,0 +1,32 @@
+VAGRANTFILE_API_VERSION = "2"
+N = 2 
+
+Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
+    (1..N).each do |machine_id|
+        config.vm.define "machine#{machine_id}" do |machine|
+            machine.vm.box = "centos/6"
+            machine.vm.hostname = "machine#{machine_id}"
```

Comment:
ugly names machine1 and machine2 :P it's better to change names to something different like centos6-wordpress etc.